packages/babel-jest/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build"
+  },
+  "references": [{"path": "../jest-types"}]
```

Comment:
add a todo about adding `babel-preset-jest` here?

Reply:
we don't actually import it, we just do `require.resolve` to pass in its path to babel.

However, I agree we should still link them